Create nxn-imitate-best-nbr

Nikocioni opened the pull request:
Il modello mi sembra simile a quello presentato sul libro, tuttavia non ottengo i risultati presentati sul libro nonostante le svariate revisioni.

Nikocioni committed:
Create nxn-imitate-best-nbr_Cioni.py

Non ottengo i risultati presentati dal capitolo del libro e dopo diverse revisioni non sono riuscito a capire il perchè

mariopaolucci commented on nxn-imitate-best-nbr_Cioni.py:
```diff
@@ -0,0 +1,174 @@
+        #Agent creation
+        agents = Agent.create_agents(model=self, n=n)
+        # Create x and y coordinates for agents
+        x = self.rng.integers(0, self.grid.width, size=(n,))
```
Questo a che serve? x viene ridefinito sotto

mariopaolucci commented on nxn-imitate-best-nbr_Cioni.py:
```diff
@@ -0,0 +1,174 @@
+class Agent(mesa.Agent):
+    def __init__(self, model):
+        super().__init__(model)
+        self.strategy = model.strategies.pop()
```
cool!

mariopaolucci reviewed (changes requested):
.. per me e' ok con queste revisioni, vediamo poi in classe come vanno i risultati

Nikocioni committed:
Update nxn-imitate-best-nbr_Cioni.py

Tolta la definizione doppia della griglia, abbassato leggermente il rumore che si è rivelato il motivo dei risultati leggermente diversi
